packages/api-specification/src/new-window-api.html

```diff
@@ -60,7 +64,12 @@
     <dt>name</dt>
     <dd>Name of the new window. Should not contain whitespace</dd>
     <dt>features (optional)</dt>
-    <dd>DOMString to set features of the new window</dd>
```

Comment:
I 100% agree, but I thought I would try to stick to the HTML5 API as much as possible with this PR.